examples/todoapp/lib/base/models/todo_model.dart

```diff
@@ -5,109 +5,135 @@
 // license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
-import 'package:equatable/equatable.dart';
 import 'package:json_annotation/json_annotation.dart';
+import 'package:realm/realm.dart';
 // ignore: implementation_imports
 import 'package:rx_bloc_list/src/models/identifiable.dart';
+
 part 'todo_model.g.dart';
+part 'todo_model.realm.dart';
 
-@JsonSerializable()
-class TodoModel with EquatableMixin implements Identifiable {
-  TodoModel({
-    required this.id,
-    required this.title,
-    this.createdAt = 0,
-    this.description = '',
-    this.completed = false,
-  });
+enum TodoModelActions { add, update, delete, none }
 
+@RealmModel()
+@JsonSerializable()
+class $TodoModel implements Identifiable {
   /// The unique identifier of the todo
   /// If the todo is not persisted yet, the id is null
-  final String? id;
+  @PrimaryKey()
+  @MapTo('_id')
+  late String? id;
 
   /// The title of the todo
   ///
   /// The title is required and must not be null
-  final String title;
+  late String title;
```

Comment:
Can't we have `final late` properties ?

Reply:
The way realm solves updating object properties requires the properties not to be final
```$TodoModel updateTodoById(String id, $TodoModel todo) {
    final realm = _realmInstance.realm;
    final result = realm.find<TodoModel>(id);

    if (result == null) {
      throw NotFoundErrorModel(message: 'Todo with id $id not found');
    }
    realm.write(() {
      result.title = todo.title;
      result.description = todo.description;
      result.completed = todo.completed;
      result.synced = todo.synced;
      result.action = todo.action;
    });
    return result;
  }